.rubocop.yml

```diff
@@ -19,16 +19,20 @@ Layout/IndentFirstHashElement:
 Layout/IndentHeredoc:
   Enabled: false
 
-Metrics/LineLength:
+Metrics/ClassLength:
   Exclude:
     - "test/**/*.rb"
```

Comment:
Excluded tests from `ClassLength` as it makes no sense there for classes with a lot of test cases.